__tests__/api.test.js

```diff
@@ -153,19 +153,92 @@ describe('/api/articles/:article_id/comments testing', () => {
     test('returns 404 Not Found when article_id is valid but the article does not exist', () => {
         return request(app)
         .get('/api/articles/99/comments')
+        .expect(404)
         .then((response) => {
             const msg = response.body.msg;
             expect(msg).toBe('Not Found')
         })
     })
 })
-describe('General error testing (404 status code)', () => {
-    test('returns a 404 Not Found message if endpoint input is invalid', () => {
+
+describe('POST /api/articles/:article_id/comments', () => {
+    test('returns 200 with the posted comment', () => {
+        return request(app)
+        .post('/api/articles/2/comments')
+        .send({
+            username: 'butter_bridge',
+            body: 'What a cool article'
+        })
+        .expect(200)
+        .then((response) => {
```

Comment:
You would be expecting a status 201 for a POST 👀

Reply:
yep, forgot about that, I fixed it just now. thanks!